src/pages/AboutPage.tsx

```diff
@@ -1,8 +1,59 @@
-import React from "react";
+import React, { useState } from "react";
+import Button from "@material-ui/core/Button";
 import { Header } from "components/Header";
 
 export const AboutPage: React.FC = () => {
+  const [aboutData, setAboutData] = useState({
+    server: "openrainbow.com",
+    telephonyVersion: "unknown",
+    systemIdentifier: "PBX7467-...",
+    CopyrightYear: "2021",
+  });
   return (
-    <Header title="Rainbow Click2Cal" subtitle="1.7.20" toPage="settings" />
+    <div>
+      <Header
+        title="Rainbow Click2Cal"
+        subtitle="Version: 1.7.20"
+        toPage="settings"
+      />
+      <div style={{ padding: "15px" }}>
```

Comment:
please remove these `style` attributes from all elements. We have to implement better styling later anyways so this will just make it more difficult for us when we add the new styling because we will have to remove them